scripts/docs/jsdocs-to-docz.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+const glob = require('glob');
+const {exec} = require('child_process');
+
+async function run(){
+  const args = process.argv.slice(2);
```

Comment:
I think this should have a default value to prevent mistakes

Reply:
the first arg is required so it should be entered by user, and the second one which is menud name, has the default value

Reply:
found the best solution for function names,
instead of trimming the submenu name, I wrote a regex that finds the function name and writes it instead of entire name and parameters of function in sub-menu.